docs/docs-ref-autogen/excel_1_1/excel/excel.chart.yml

```diff
@@ -1,8 +1,10 @@
 ### YamlMime:UniversalReference
 items:
   - uid: excel.Excel.Chart
-    summary: |-
-      Represents a chart object in a workbook.
+    summary: >-
+      Represents a chart object in a workbook. To learn more about the Chart object model, see [Work with charts using
+      the Excel JavaScript API](https://docs.microsoft.com/office/dev/add-ins/excel/excel-add-ins-charts)<!-- -->.
```

Comment:
Do we have any concerns about articles like this using APIs that were introduced in later requirement sets?

Reply:
I don't think so. We could consider versioned conceptual docs, but that feels like a lot of effort for a low impact scenario. Those Excel docs are also relatively scenario-driven, so if you're looking to work with PivotTables, you'll need the versions of those APIs.